defaults/main.yml

```diff
@@ -38,13 +38,15 @@ openhpc_slurmdbd_mysql_username: slurm
 openhpc_enable:
   control: false
   batch: false
+  login: true
```

Comment:
Just thinking that we might want to support the SRV records as well. Just wondering if we need a flag that is more explicit like: `openhpc_login_openhpc_enable: true`?

Reply:
Thought about it some more, maybe you are right, it will just be simpler if just always configure slurmd on the login nodes. I guess there isn't much benefit form not doing so?

Reply:
Hmm. I guess with a separate flag it'd also need `openhpc_enable.batch` setting (either by the user, or in the code). It feels like either way the docs for that are going to get confusing - it'll feel like why isn't that parallel to the control/batch flags? There merit in not running services if you don't need them though. Guess we could just call it `openhpc_enable.slurmd_login`?

Reply:
I guess you always override `ohpc_slurm_services` to disable slurmd. Might be nice if there was some variable like `ohpc_slurm_services_login`, so you didn't have to override the whole dict, but that could be a future addition.